static/app/views/dashboards/widgetBuilder/widgetBuilder.tsx

```diff
@@ -1138,6 +1149,10 @@ function WidgetBuilder({
 
 export default withPageFilters(withTags(WidgetBuilder));
 
+const TitleInput = styled(InputField)`
+  padding: 0;
```

Comment:
I know I already commented on a change here, but I realized that the right side should have a bit of padding to align the elements :)

Can we change this to:

```suggestion
  padding: 0 ${space(2)} 0 0;
```